### DISCUSSION TITLE
Can the Burger ordering example make multiple burgers based on an amount that is free input

### DISCUSSION THREAD

**Discussion by rbel-io** [charmbracelet/huh#153] [Q&A]
I just tinkering around getting more familiar with huh and the possibilities.
Specially the [burger](https://github.com/charmbracelet/huh/blob/main/examples/burger/main.go) intro example shows allot of possibilities, now I wonder what would be the approach if we order multiple burgers in one run of the form? 

E.g.:
1. How many burgers you want to order? 5 (could be any number form should "scale" automatically)
2. Burger number 1:  (shows the original form)
3. Burger number 2: (shows the original form)
4. Etc
5. Etc
6. Etc
7. Summary with all the burgers.

**Comment by maaslalani** [comment]
Hey there! Yes, you can certainly do this!

How I would approach this is to create a form to prompt the user how many burgers they want. Like so:

```go
var numberOfBurgers int
_ = huh.NewSelect[int](huh.NewOptions(1, 2, 3, 4, 5)...).Value(&numberOfBurgers).Run()
```

Then, show the form `n` times:

```go
var burgersList []Burger
var burger Burger
for i := 0; i < numberOfBurgers; i++ {
    // Run burger form as normal.

    // Store this finished burger.
    burgersList = append(burgersList, burger)
}

// Print out summary of burger
for _, b := range burgerList {
    // Print burger
}
```
